hlx_statics/blocks/inline-nested-alert/inline-nested-alert.js

```diff
@@ -0,0 +1,129 @@
+import { createTag, decorateAnchorLink } from "../../scripts/lib-adobeio.js";
+
+function addInfoSvg() {
+    return `
+        <svg
+            class="spectrum-Icon info-NestedAlertIcon nested-svg"
+            <path
```

Comment:
This is the same as https://github.com/adobe/adobe-io-website/blob/fresh-spectrum-less/hlx_statics/blocks/inlinealert/inlinealert.js#L12-L17.

I think we can extract this to a util and then reference in both InlineAlert and InlineNestedAlert blocks